src/devices/Ads1115/Ads1115.cs

```diff
@@ -104,14 +251,235 @@ private void SetConfig()
             _i2cDevice.Write(writeBuff);
 
             // waiting for the sensor stability
-            Thread.Sleep(10);
+            WaitWhileBusy();
+
+            if (_deviceMode == DeviceMode.Continuous)
+            {
+                // We need to wait two cycles when changing the configuration in continuous mode,
+                // otherwise we may be getting a value from the wrong input
+                _i2cDevice.Write(writeBuff);
+                WaitWhileBusy();
+                configHi &= 0xFE; // Clear last bit
```

Comment:
NIT: usually for things like this we just do `&= ~0b01` which makes it easier to read.

Reply:
The ugly thing is that that requires an extra cast, because `~0b01` is -2 and therefore a signed value which cannot be directly applied to a `byte`.

Reply:
Yeah you are right, I  missed this was a byte and not an int. In that case, I would probably just do `configHi &= 0b_1111_1110;` Since I guess that makes it more readable than having to convert HEX to Binary on my mind 😄